h2o-py/tests/testdir_jira/pyunit_pubdev_7252_expose_original_training_frame.py

```diff
@@ -0,0 +1,92 @@
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.gbm import H2OGradientBoostingEstimator
+from h2o.estimators.isolation_forest import H2OIsolationForestEstimator
+from h2o.estimators.random_forest import H2ORandomForestEstimator
+from h2o.estimators.xgboost import H2OXGBoostEstimator
+
+
+
+def test_expose_original_training_frame():
+    airlines= h2o.import_file(path=pyunit_utils.locate("smalldata/airlines/allyears2k_headers.zip"))
+
+    # convert columns to factors
+    airlines["Year"]= airlines["Year"].asfactor()
+    airlines["Month"]= airlines["Month"].asfactor()
+    airlines["DayOfWeek"] = airlines["DayOfWeek"].asfactor()
+    airlines["Cancelled"] = airlines["Cancelled"].asfactor()
+    airlines['FlightNum'] = airlines['FlightNum'].asfactor()
+
+
+    # set the predictor names and the response column name
+    predictors = ["Origin", "Dest", "Year", "UniqueCarrier", "DayOfWeek", "Month", "Distance", "FlightNum"]
+    response = "IsDepDelayed"
+    predictorsAndResponse = predictors
+    predictorsAndResponse.append(response)
```

Comment:
this modifies the predictors list as well, I think what you want to do is `predictors + [response]`